getting_started.rst

```diff
@@ -3,11 +3,14 @@
 Getting Started
 ===============
 
-Let's get started with an example: We are going to create an MD5 encrypter app that uses one of the most popular C++ libraries: Poco_.
+Let's get started with an example: We are going to create an MD5 hash calculator app that uses one of the
+most popular C++ libraries: Poco_.
 
-We'll use CMake as build system in this case but keep in mind that Conan **works with any build system** and is not limited to using CMake.
+We'll use CMake as build system in this case but keep in mind that Conan **works with any build
+system** and is not limited to using CMake.
 
-Make sure you are running the latest Conan version. Read the :ref:`Conan update<conan_update>` section get more information.
+Make sure you are running the latest Conan version. Read the :ref:`Conan update<conan_update>`
+section get more information.
 
 An MD5 Encrypter using the Poco Libraries
```

Comment:
This should say "hash calculator" too, but we need to preserve the previous link (add it to the `conf.py` file)

Reply:
We'll leave the conf.py as the link is an anchor and won'tfail, but I've changed the title in the other pages where it is referenced.